requirements.txt

```diff
@@ -5,3 +5,4 @@
 Adafruit-Blinka
 adafruit-circuitpython-busdevice
 pyserial
+typing-extensions~=4.0
```

Comment:
Don't forget to now add `circuitpython_typing`!

Reply:
oh yeah, that's in my notes on using Writeablebuffer for sure.....